File: Exercicio-03.c
#include <stdio.h>

int main(void) {

  int number, denominator, resul;

  printf("Digite o numerador: ");
  scanf("%d", &number);

  printf("Digite o denominador: ");
  scanf("%d", &denominator);

  resul = number % denominator;

  printf("O resto da divisão é: %d", resul);
  
  return 0;
}

File: Exercicio-03.java
import java.util.Scanner;

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    int number, denominator, resul;

    System.out.printf("Digite o numerador: ");
    number = sc.nextInt();

    System.out.printf("Digite o denominador: ");
    denominator = sc.nextInt();
    
    resul = number % denominator;
    
    System.out.println("O resto da divisão é: " + resul);
  }
}
